src/applications/vaos/components/calendar/CalendarWeekdayHeader.jsx

```diff
@@ -19,7 +19,7 @@ export default function CalendarWeekdayHeader({ showFullWeek = false }) {
   return (
     <div role="rowgroup">
       <div
-        className="vaos-calendar__weekday-container vads-u-display--flex vads-u-justify-content--space-between"
+        className="vaos-calendar__weekday-container vads-u-display--flex vads-u-justify-content--space-around"
```

Comment:
Great call on `vads-u-justify-content--space-around`! Where did you find documentation on that class name?

Reply:
@jenniemc ...I just used Chrome dev tools inspect tab to search for the class name